app/jobs/email.py

```diff
@@ -6,13 +6,51 @@
 from app import Q
 
 
+class EmailConfig:
+    def __init__(self):
+        self.smtp_username = current_app.config["MAIL_USERNAME"]
+        self.smtp_password = current_app.config["MAIL_PASSWORD"]
+        self.host = current_app.config["MAIL_SERVER"]
+        self.port = current_app.config["MAIL_PORT"]
+        self.tls = current_app.config["MAIL_USE_TLS"]
+
+
+@Q.job(timeout=60000)
+def send_beta_backlog_notification_email(email_address):
+    config = EmailConfig()
+    msg = MIMEMultipart("alternative")
```

Comment:
What if we moved all of this configuration into Email classes and then gave those classes the responsibility to send the correct email.   That would let us move "Emails" into a different object hierarchy so we're not storing it all inside the body of these job functions.